lib/pages/chat/chat.dart

```diff
@@ -1930,7 +1931,7 @@ class ChatController extends State<Chat>
     pinnedMessageScrollController.dispose();
     onUpdateEventStreamSubcription?.cancel();
     keyboardVisibilitySubscription?.cancel();
-
+    InViewNotifierListCustom.of(context)?.dispose();
```

Comment:
it still dispose `inview` well?

Reply:
yes, here is where its reference.
<img width="747" alt="Screenshot 2024-05-13 at 17 53 25" src="https://github.com/linagora/twake-on-matrix/assets/43041967/bec16099-e6ce-45ce-9b0f-ce8007af34a7">